exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -1564,3 +1564,40 @@ func (ku *Kucoin) GetFuturesContractDetails(ctx context.Context, item asset.Item
 	}
 	return resp, nil
 }
+
+// UpdateOrderExecutionLimits updates order execution limits
+func (ku *Kucoin) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
```

Comment:
Please add this to `Run()` like all other `UpdateOrderExecutionLimits()` implementations